Dockerfile

```diff
@@ -11,9 +11,29 @@ RUN set -ex && \
     apt-get upgrade -y && \
     apt-get install git -y && \
     apt-get install -y \
-        python3 \
```

Comment:
Previously, I think we've been grabbing the latest python3 version every time we push a new version of this image. This will also pin the python3 version to 3.9.2 (I think blanket upgrading to 3.11 at this point could break some of our builds that rely on this agent)

Reply:
Is it an option to install the latest `3.9.x` version? Like `pyenv install 3.9` or something?

Reply:
Yes, good call! Updated to just `3.9` instead of `3.9.2`!

Reply:
I guess it makes it slightly less deterministic than using the full version, but it's still a step in the right direction.